### PR TITLE
Fixes for v2

### PR DESCRIPTION
**Pull request by connor-ms**

### CODE REVIEW

File: src/api/recipe.py
Comment (sethlangel) [inline src/api/recipe.py:1]: Why is author_id a float?

Comment (sethlangel) [inline src/api/recipe.py:1]: Why is ingredient_id a float?

**Review by sethlangel — CHANGES_REQUESTED** [review]

File: src/api/recipe.py
Comment (connor-ms) [reply src/api/recipe.py:1]: Because I'm a stupid little dum dum

**Review by sethlangel — APPROVED** [review]
Looks better!

**Pull request merged**